diffusion/models/layers.py

```diff
@@ -14,8 +14,10 @@
 except:
     pass
 
+T = TypeVar('T', bound=nn.Module)
```

Comment:
This is a private a typevar, should be prefixed with a underscore.

Reply:
Good point! Fixed now!